.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: Build Android and iOS
+on:
+  push:
+    branches:
+      - master
+jobs:
+  install-and-test:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Install npm dependencies
```

Comment:
How does it know what working  directories to run the `npm` commands @Flexicoder ?

Does it need a `working-directory:` somewhere?

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions

Reply:
It will just default to `./` - as these actions on running on their servers, does it matter? Its not going to affect the repository?

Reply:
Yea it's all relative from the repo root

Reply:
Wont this fail though? Doesnt the action need to know that `npm install` needs to run:

> .\codebase\Dossier

There is no `package.json` within the root. That was my confusion 🥰

Reply:
As per WhatsApp, let me see what happens 👯‍♂️